modules/chapter1/pages/section2.adoc

```diff
@@ -1,3 +1,40 @@
-= Section 2
+= Architecture of Red{nbsp}Hat OpenShift Data Science
```

Comment:
There should be a way to add a shorter text to the navigation bar - see the ROOT module home page. You should add :navtitle: Architecture to this file

Reply:
done